tools/nitrogfx/main.c

```diff
@@ -288,7 +288,7 @@ void HandleNtrToPngCommand(char *inputPath, char *outputPath, int argc, char **a
             if (!ParseNumber(argv[i], NULL, 10, &options.palIndex))
                 FATAL_ERROR("Failed to parse palette index.\n");
 
-            if (options.width < 1)
+            if (options.palIndex < 1)
```

Comment:
would be appreciated if this was added to red031000/nitrogfx as well

Reply:
I made a PR for this back in September: https://github.com/red031000/nitrogfx/pull/6